Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_First_SSO_Access.yml

```diff
@@ -0,0 +1,1906 @@
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -110,
+          "y": -1080
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "160":
+    id: "160"
+    taskid: 36cba334-80b3-45b9-8f46-d04ab74bc01a
+    type: condition
+    task:
+      id: 36cba334-80b3-45b9-8f46-d04ab74bc01a
+      version: -1
+      name: Should the Account be disabled automatically?
+      description: Checks if the account can be disabled and resets the account's password automatically based on the input's value. (inputs.AutomaticallyBlockAccount)
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "79"
+      "yes":
+      - "143"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.AutomaticallyBlockAccount
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 917.5,
+          "y": 2230
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "111_112_yes": 0.4,
+      "111_115_#default#": 0.11,
+      "116_120_yes": 0.56,
+      "147_146_yes": 0.63,
+      "153_72_yes": 0.43,
+      "153_76_#default#": 0.1,
+      "158_150_yes": 0.49,
+      "159_147_yes": 0.44,
+      "160_143_yes": 0.56,
+      "34_155_Suspicious": 0.4,
+      "34_72_Malicious": 0.23,
+      "34_76_Non-malicious": 0.18,
+      "34_88_Suspicious - Ask the Manager": 0.62,
+      "88_100_yes": 0.51,
+      "88_155_#default#": 0.34
+    },
+    "paper": {
+      "dimensions": {
+        "height": 5125,
+        "width": 2230,
+        "x": -520,
+        "y": -1850
+      }
+    }
+  }
+inputs:
+- key: AutomaticallyIsolateEndpoint
+  value:
+    simple: "False"
+  required: false
+  description: Whether to isolate the endpoint automatically.
+  playbookInputQuery:
+- key: NumOfXdrAlertsThreshold
+  value:
+    simple: "3"
+  required: false
+  description: |-
+    The 'XDR related alerts' threshold to determine the 'Check for XDR related alerts' check result in the 'Cortex XDR - First SSO Access - Set Verdict' sub-playbook.
+    The default value is '3'.
+  playbookInputQuery:
+- key: FailedlogonUserThreshold
+  value:
+    simple: "30"
+  required: false
+  description: |-
+    The 'Failed login' threshold to determine the 'Check for massive failed logon' check result in the 'Cortex XDR - First SSO Access - Set Verdict' sub-playbook.
```

Comment:
same comment here

Reply:
Done.
changed to:
This is the minimum threshold for failed login attempts by the user.
example: If this input is set to '30', and the 'Cortex XDR - First SSO Access - Set Verdict' sub-playbook has found 31 failed login attempts - It will classify this check as suspicious activity.
The default value is '30'.